vendor/github.com/containers/image/docker/archive/dest.go

```diff
@@ -19,15 +19,31 @@ func newImageDestination(ctx *types.SystemContext, ref archiveReference) (types.
 	if ref.destinationRef == nil {
```

Comment:
vendoring updates should be done using vndr and an update to the commit in vendor.conf